### ISSUE TITLE
Failed at Building

### ISSUE THREAD

**Issue by KolpixTV** [ddnet/ddnet#4266]
```
/usr/bin/ld: CMakeFiles/DDNet.dir/src/engine/client/client.cpp.o: in function `CClient::MapDownloadTotalsize() const':
client.cpp:(.text._ZNK7CClient20MapDownloadTotalsizeEv[_ZNK7CClient20MapDownloadTotalsizeEv]+0x30): undefined reference to `__atomic_load_8'
/usr/bin/ld: CMakeFiles/DDNet.dir/src/engine/client/client.cpp.o: in function `CClient::MapDownloadAmount() const':
client.cpp:(.text._ZNK7CClient17MapDownloadAmountEv[_ZNK7CClient17MapDownloadAmountEv]+0x30): undefined reference to `__atomic_load_8'
/usr/bin/ld: CMakeFiles/DDNet.dir/src/engine/client/http.cpp.o: in function `CRequest::ProgressCallback(void*, double, double, double, double)':
http.cpp:(.text+0x524): undefined reference to `__atomic_store_8'
/usr/bin/ld: http.cpp:(.text+0x534): undefined reference to `__atomic_store_8'
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/DDNet.dir/build.make:1652: DDNet] Error 1
make[1]: *** [CMakeFiles/Makefile2:374: CMakeFiles/DDNet.dir/all] Error 2
make: *** [Makefile:152: all] Error 2

```

**Comment by KolpixTV** [comment]
fixed
![image](https://user-images.githubusercontent.com/68741965/139578030-dd8d2aa5-8ef8-484f-a5d2-25a7ebe78ee4.png)
(CMakeLists.txt)

**Comment by def-** [comment]
I guess we should link to atomic. I also found this: https://gitlab.kitware.com/cmake/cmake/-/issues/21174

**Comment by Jupeyy** [comment]
probably fixed in #4556 . Reopen if not
